Nanoleaf.Client/Nanoleaf.Client/Models/Responses/Layout.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.ComponentModel;
+using Newtonsoft.Json;
+
+namespace Nanoleaf.Client.Models.Responses {
+	/// <summary>
+	/// Layout response returned from a layout request
+	/// </summary>
+	[Serializable]
+	public class Layout {
+		/// <summary>
+		/// Number of panels
+		/// </summary>
+		[JsonProperty] public int NumPanels { get; set; }
```

Comment:
Please use the existing code design. Attributes should be on a separate line from a property.

Reply:
Fixed with 21f16ef, f39c588.